### PR TITLE
Enhanced NewOrder Filtering to prevent timeout

### PR DESCRIPTION
**Pull request by harmlessprince**

### CODE REVIEW

File: app/NewOrder.php [hunk 1]
```diff
@@ -170,39 +170,39 @@ public function getOrderPaymentMethodAttribute()
...
-            "order_payment_method" => $this->order_payment_method,
-            "customer" => $this->customer,
-            "order_date" => $this->order_date,
-            "owner" => $this->owner->full_name ?? '',
-            "sales_type" => $this->salesCategory ?? '',
-            "branch_id" => $this->branch->id,
-            "owner_id" => $this->owner->id
-        ];
-    }
+    // public function toArray()
```
Comment (teebarg) [inline app/NewOrder.php:173]: Why did you comment this out?

File: app/Services/NewOrdersReportService.php [hunk 2]
```diff
@@ -84,3 +94,5 @@ private  function getComparismOfAltaraPayVsAltaraCash($totalAltaraCash, $totalAl
         ];
     }
 }
+// SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
```
Comment (teebarg) [inline app/Services/NewOrdersReportService.php:97]: do we still need this here?

File: routes/api.php [hunk 3]
```diff
@@ -118,11 +118,12 @@
     Route::get('/feedbacks/export', 'FeedbackController@export');
     Route::post('/recommendation', 'AmortizationController@recommend');
 
-    Route::get('/order/reports', 'NewOrderController@report');
+    // Route::get('/order/reports', 'NewOrderController@report');
```
Comment (teebarg) [inline routes/api.php:121]: why was this commented out?

File: app/NewOrder.php (hunk 1 above)
Comment (harmlessprince) [reply app/NewOrder.php:173]: will be cleaned, did that while I was debugging

File: routes/api.php (hunk 3 above)
Comment (harmlessprince) [reply routes/api.php:121]: will be cleaned up as well

**Pull request closed without merging**